hybrid/dispatch/power_sources/power_source_dispatch.py

```diff
@@ -67,6 +69,8 @@ def initialize_parameters(self):
     def update_time_series_parameters(self, start_time: int):
         n_horizon = len(self.blocks.index_set())
         generation = self._system_model.value("gen")
+        if isinstance(self._system_model, Pvsamv1.Pvsamv1):
+            generation = tuple(np.clip(generation, 0, None))   # remove negative generation (i.e., load)
```

Comment:
@dguittet I had to do this here as the _system_model is not a detailed_pv_plant here but rather the pvsamv1 pysam model

Reply:
I over-rode it in `PvDispatch`